index.js

```diff
@@ -17,20 +25,35 @@ module.exports.file = options => {
 			throw new Error('The `name` and `extension` options are mutually exclusive');
 		}
 
-		return path.join(module.exports.directory(), options.name);
+		return path.join(getPath(), options.name);
 	}
 
 	return getPath() + '.' + options.extension.replace(/^\./, '');
 };
 
-module.exports.directory = () => {
+module.exports.directory = getPath;
+
+module.exports.clean = () => {
+	const deletedDirectories = del.sync(cacheDirectories.map(directory => directory + '/**'), {force: true});
+	cacheDirectories = [];
+	return deletedDirectories;
```

Comment:
You need to make sure this never throws.

It's only not ideal that this is synchronous. We might need an async version too... Unless you can think of a better alternative?